modules/using-oci-containers-for-model-storage.adoc

```diff
@@ -0,0 +1,209 @@
+====
+
+== Deploying a model stored in an OCI image from a public repository
+
+[NOTE]
+====
+By default in KServe, models are exposed outside the cluster and not protected with authorization. 
+====
+
+. Create a namespace to deploy the model:
++
+[source]
+----
+oc new-project oci-model-example
+----
++
+
+ifndef::upstream[]
+. Use the {productname-short} Applications project `kserve-ovms` template to create a `ServingRuntime` resource and configure the OpenVINO model server in the new namespace:
++
+[source]
+----
+oc process -n redhat-ods-applications -o yaml kserve-ovms | oc apply -f -
+----
+endif::[]
+ifdef::upstream[]
+. Use the {productname-short} project `kserve-ovms` template to create a `ServingRuntime` resource and configure the OpenVINO model server in the new namespace:
++
+[source]
+----
+oc process -n opendatahub -o yaml kserve-ovms | oc apply -f -
+----
+endif::[]
++
+--
+.. Verify that the `ServingRuntime` has been created with the `kserve-ovms` name:
++
+[source]
+----
+oc get servingruntimes
+
+NAME          DISABLED   MODELTYPE     CONTAINERS         AGE
+kserve-ovms              openvino_ir   kserve-container   1m
+----
+--
++
+. Create an `InferenceService` YAML resource with the following values:
++
+[source]
+----
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: sample-isvc-using-oci
+spec:
+  predictor:
+    model:
+      runtime: kserve-ovms # Ensure this matches the name of the ServingRuntime resource
+      modelFormat:
+        name: onnx
+      storageUri: oci://quay.io/<user_name>/<repository_name>:<tag_name>
+----
++
+[IMPORTANT]
+====
+The `ServingRuntime` and `InferenceService` configurations do not set any resource limits.
+====
+
+.Verification
+After you create the `InferenceService` resource, KServe deploys the model stored in the OCI image referred to by the `storageUri` field. Check the status of the deployment with the following command:
+[source]
+----
+oc get inferenceservice
+
+NAME                    URL                                                       READY   PREV   LATEST   PREVROLLEDOUTREVISION   LATESTREADYREVISION                     AGE
+sample-isvc-using-oci   https://sample-isvc-using-oci-oci-model-example.example   True           100                              sample-isvc-using-oci-predictor-00001   1m
+----
+
+== Deploying a model stored in an OCI image from a private repository
+
+To deploy a model stored from a private OCI repository, you must configure an image pull secret. For more information about creating an image pull secret, see link:https://docs.openshift.com/container-platform/latest/openshift_images/managing_images/using-image-pull-secrets.html[Using image pull secrets^].
+
+. Follow the steps in the previous section for deploying a model. However, when creating the `InferenceService` in step 3, specify your pull secret in the
+`spec.predictor.imagePullSecrets` field:
```

Comment:
Minor comment, but wondering if you added the newline here unintentionally? Assuming it would be _specify your pull secret in the`spec.predictor.imagePullSecrets` field:_ , i.e, all in the same line.

Reply:
Great catch, thank you!